examples/change_case/workflow-test.yml

```diff
@@ -38,6 +38,15 @@
 # Enable to print yet more information
 #enable_debug: True
 
+# Maximum number of retries after a connection failure
+# max_retries: 10
+
+# Delay between consecutive retries
```

Comment:
Are these delays in seconds? Might be good to specify it.

Reply:
Fixed by https://github.com/phnmnl/wft4galaxy/pull/41/commits/31b6cf1abeef34eca146afc5b2f114afcbcd4f8c